code/parachain/frame/staking-rewards/proptest-regressions/test/mod.txt

```diff
@@ -0,0 +1,10 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc e9c54402dec866a96921319f195004cdce8fe2d93911459a42103768990eeea6 # shrinks to network_id = 1, amm_id = 0, start_block = 1
+cc f60478cbb9380be7fea6c421e8ce2ab80edabce698b465c9669145deb780b19f # shrinks to amount = 170141183460469231731687303715884105228
+cc da2d5615192fd7056f17bb99e4904d6a0d385252d7a2fb795e9071a55130be39 # shrinks to amount = 336913234575186597488489710328483377680
+cc 3ea68462befd66841315d4b585ba1f548f7cdde9e6184766818c0ca93b874b5e # shrinks to parts = 990000
```

Comment:
Did we intend to commit this file?

Reply:
I would guess that we don't, but there are a few files like this already commited. I will remove it anyway.